nomic/project.py

```diff
@@ -963,11 +967,62 @@ def get_projection_id():
 
     def __repr__(self):
         m = self.meta
-        return f"Nomic project: <{m}>"
+        return f"AtlasProject: <{m}>"
 
     def __str__(self):
         return "\n".join([str(projection) for index in self.indices for projection in index.projections])
 
+
+    def get_data(self, ids: List[str]) -> List[Dict]:
+        '''
+        Retrieve the contents of the data given ids
+
+        Args:
+            ids: a list of datum ids
+
+        Returns:
+            A list of dictionaries corresponding
+
+        '''
+
+        if not isinstance(ids, list):
+            raise ValueError("You must specify a list of ids when getting data.")
+
+        response = requests.post(
+            self.atlas_api_path + "/v1/project/data/get",
+            headers=self.header,
+            json={'project_id': self.id, 'datum_ids': ids},
+        )
+
+        if response.status_code == 200:
+            return [item for item in response.json()['datums']]
+        else:
+            raise Exception(response.json())
+
+    def delete_data(self, ids: List[str]) -> bool:
+        '''
+        Deletes the specified datums from the project.
+
+        Args:
+            ids: A list of datum ids to delete
+
+        Returns:
```

Comment:
This returns `True`. Perhaps it should return `None` on successful completion--that's more the Unix standard.